### PR TITLE
refactor(platform-browser-dynamic): Removed TestComponentBuilder from…

### PR DESCRIPTION
**Pull request by hansl**
**Please check if the PR fulfills these requirements**
- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit-message-format
- [X] Tests for the changes have been added (for bug fixes / features)

---

**What kind of change does this PR introduce?** (check one with "x")
- [X] Refactoring (no functional changes, no api changes)

---

**What is the current behavior?** (You can also link to an open issue here)
Tests were using a deprecated API (`TestComponentBuilder`).

---

**What is the new behavior?**
Tests now use `TestBed`.

---

**Does this PR introduce a breaking change?** (check one with "x")
- [X] No

### CODE REVIEW

File: modules/@angular/platform-browser-dynamic/test/resource_loader/resource_loader_cache_spec.ts [hunk 1]
```diff
@@ -18,55 +18,55 @@ import {setTemplateCache} from './resource_loader_cache_setter';
 
 export function main() {
   describe('CachedResourceLoader', () => {
-    var xhr: CachedResourceLoader;
+    var resourceLoader: CachedResourceLoader;
```
Comment (vicb) [inline modules/@angular/platform-browser-dynamic/test/resource_loader/resource_loader_cache_spec.ts]: nitpick: `let`

**Comment by vicb** [comment]
LGTM, one nitpick

**Comment by hansl** [comment]
I just renamed the variable using a refactoring tool, sorry :)

**Comment by vicb** [comment]
Maybe you should refactor the refactoring tool ?

**Pull request merged**